src/ptvsd/adapter/launchers.py

```diff
@@ -66,7 +66,7 @@ def spawn_launcher():
             if log.log_dir is not None:
                 env[str("PTVSD_LOG_DIR")] = compat.filename_str(log.log_dir)
             if log.stderr.levels != {"warning", "error"}:
-                env[str("PTVSD_LOG_STDERR")] = " ".join(log.log_stderr)
```

Comment:
I think that you should only put bytes in Py2 in the env (as this file has a `from __future__ import unicode_literals`, the right side may end up with a unicode, so, I guess it needs to be `str(" ".join(log.stderr.levels))`) -- or maybe it can be fixed right before passing the env to `Popen`.